hudi-hadoop-common/src/main/java/org/apache/parquet/avro/HoodieAvroReadSupport.java

```diff
@@ -58,6 +61,40 @@ public ReadContext init(Configuration configuration, Map<String, String> keyValu
     return new ReadContext(requestedSchema, readContext.getReadSupportMetadata());
   }
 
+  public ReadContext init(ParquetConfiguration configuration, Map<String, String> keyValueMetaData, MessageType fileSchema) {
```

Comment:
nit: add `@Override` to mark that we need to override the default behavior similar to the other `init` method